pkg/elevate/elevate_test.go

```diff
@@ -39,9 +39,7 @@ var fakeAPIConfig = api.Config{
 		},
 	},
 	AuthInfos: map[string]*api.AuthInfo{
-		"anonymous": {
-			LocationOfOrigin: "England",
```

Comment:
This is a good catch, it always create unwanted file :)

Reply:
Yeah this one took a bit. There's some code in the k8s kubeconfig package that basically writes the file with the value from `LocationOfOrigin` in the current working directory if you don't tell it otherwise. The real fix was (unfortunately) relying on setting the temporary value for `KUBECONFIG` in the tests. In the future I can come back through here and wire in a fake filesystem which will make it much easier to make assertions without the risk of touching local files at all.